<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TimerPage</class>
 <widget class="QWidget" name="TimerPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>440</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>350</width>
    <height>440</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>350</width>
    <height>440</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border-radius:5px;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="CentralWidget">
     <property name="styleSheet">
      <string notr="true">QFrame{
	background-color: rgb(240, 240, 240);
	border-bottom-left-radius:0px;
	border-bottom-right-radius:0px;
	border-top-left-radius:0px;
	border-top-right-radius:0px;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QFrame" name="TimerFrame">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <widget class="QWidget" name="Progress_Front" native="true">
         <property name="geometry">
          <rect>
           <x>32</x>
           <y>48</y>
           <width>270</width>
           <height>270</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>270</width>
           <height>270</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QWidget{
   border-radius: 135px;
   background-color: rgb(238, 213, 214);
}</string>
         </property>
         <widget class="QPushButton" name="TimerButton">
          <property name="geometry">
           <rect>
            <x>5</x>
            <y>5</y>
            <width>260</width>
            <height>260</height>
           </rect>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>190</width>
            <height>190</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
	border:none;
	border-radius:130px;
	background-color: rgb(240, 240, 240);
	color :  rgb(255, 100, 100);
	font-size : 60px;
	font-weight : 100;
}
QPushButton:Click{

	border:none

}</string>
          </property>
          <property name="text">
           <string>00:00</string>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="Progress_Back" native="true">
         <property name="geometry">
          <rect>
           <x>32</x>
           <y>48</y>
           <width>270</width>
           <height>270</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>270</width>
           <height>270</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
        </widget>
        <widget class="QLabel" name="TimerStateLabel">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>110</y>
           <width>332</width>
           <height>42</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>332</width>
           <height>42</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel{
	font-size : 25px;
	background-color : none;
}</string>
         </property>
         <property name="text">
          <string>pomordor</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QLabel" name="SetCountLabel">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>240</y>
           <width>332</width>
           <height>30</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>332</width>
           <height>30</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel{
	font-size : 15px;
  font-weight : 500;
	background-color : none;
}</string>
         </property>
         <property name="text">
          <string>Set Count : 0 / 3</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <zorder>Progress_Back</zorder>
        <zorder>Progress_Front</zorder>
        <zorder>SetCountLabel</zorder>
        <zorder>TimerStateLabel</zorder>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="ButtonFrame">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>50</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="spacing">
          <number>20</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <spacer name="Play_HLSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>131</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="PlayButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
	Border:none;
	border-radius:8px;
	background-color: rgb(255, 100, 100);
}
QPushButton:Hover{
	background-color: rgba(255, 135, 135, 200);
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../../assets/files.qrc">
             <normaloff>:/20x20/icons/20x20/cil-media-play.png</normaloff>:/20x20/icons/20x20/cil-media-play.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="Play_HRSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>131</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../assets/files.qrc"/>
 </resources>
 <connections/>
</ui>
